### PR TITLE
Added api for Message of the day, fixed some SASS compile errors

### PR DESCRIPTION
**Pull request by Fidaman**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/Camelot-Unchained-Client-Library/pull/26" target="_blank">ReviewNinja

File: src/restapi/RestAPI.ts [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 import {Promise} from 'es6-promise';
 import * as RestClientLegacy from './RestClientLegacy';
+import * as RestClient from './RestClient';
+
+export function getMessageOfTheDay()
+{
```
Comment (Mehuge) [inline src/restapi/RestAPI.ts]: opening brace should not have a newline before them rather a space

https://github.com/airbnb/javascript#blocks--cuddled-elses (closest rule I could find that implies this rule)

**Comment by Fidaman** [comment]
oops that last checkin was done with my work account by mistake. That's what I get for context switching.

**Comment by codecorsair** [comment]
:+1:

**Pull request merged**